force-app/main/default/classes/HOT_ResourceAbsenceCreateService.cls

```diff
@@ -16,12 +16,12 @@ public without sharing class HOT_ResourceAbsenceCreateService implements Databas
             this.timeScope = Datetime.now().addHours(-1);
         }
         System.debug('timeScope: ' + timeScope);
-        this.query = 'SELECT Id, OwnerId, StartDateTime, EndDateTime, ShowAs, Subject FROM Event WHERE RecordType.DeveloperName = \'HOT_Events\' AND WhatId = null AND LastModifiedDate > :timeScope AND ShowAs != \'Free\'';
+        this.query = 'SELECT Id, OwnerId, StartDateTime, EndDateTime, ShowAs, Subject FROM Event WHERE RecordType.DeveloperName = \'HOT_Events\' AND What.Type != \'SeriveAppointment\' AND LastModifiedDate > :timeScope AND ShowAs != \'Free\'';
```

Comment:
Her er det en skrivefeil (gjelder flere steder): ServiceAppointment. Må også legge til `AND What.Type != 'ResourceAbsence'`